kong/db/migrations/core/005_120_to_130.lua

```diff
@@ -1,3 +1,88 @@
+
+local function add_upstream_algorithm_to_postgress(connector)
+  assert(connector:connect_migrations())
+  local select_query = [[
+    SELECT * FROM "upstreams";
+  ]]
+
+  for upstream, err in connector:iterate(select_query) do
+    if err then
+      return nil, err
+    end
+
+    if upstream.algorithm ~= nil and upstream.algorithm ~= ngx.null
+      and type(upstream.algorithm) == "string" and upstream.algorithm:len() > 0 then
+      goto continue
+    end
+    local algorithm
+    if upstream.hash_on == "none" then
+      algorithm = "round-robin"
+    else
+      algorithm = "consistent-hashing"
+    end
+    local update_query = string.format([[
+      UPDATE "upstreams"
+      WHERE "id" = '%s';
+      SET "algorithm" = '%s'
+    ]], algorithm, upstream.id)
+
+    local _, err = connector:query(update_query)
+    if err then
+      return nil, err
+    end
+
+    ::continue::
+  end
+end
+
+
+local function add_upstream_algorithm_to_cassandra(connector)
+  local cassandra = require "cassandra"
+  local coordinator = assert(connector:connect_migrations())
+
+  local select_query = [[
+    SELECT * FROM upstreams;
+  ]]
+  local update_query = [[
+    UPDATE upstreams
+    SET algorithm = ?
+    WHERE id = ?
+  ]]
```

Comment:
A minor complaint, but throughout this file, we write the SQL/CQL queries in local variables, but I think it makes the code less readable, especially when the query is defined many lines above its use (e.g. here with `update_query`). We could just inline those queries in the `iterate()` and `query()` calls arguments.